main.tf

```diff
@@ -1,11 +1,20 @@
 locals {
+  tenant_id = length(var.tenant_id) > 0 ? var.tenant_id : data.azurerm_subscription.primary.tenant_id
```

Comment:
We must update this output: https://github.com/lacework/terraform-azure-config/blob/main/output.tf#L17-L20